src/borg/helpers/passphrase.py

```diff
@@ -109,13 +110,15 @@ def verification(cls, passphrase):
             retry=True,
             env_var_override="BORG_DISPLAY_PASSPHRASE",
         ):
-            pw_msg = f"""
+            pw_msg = textwrap.dedent(
+                f"""
```

Comment:
BTW, this introduces a newline at the start of the string.

Sometimes that is useful, but if it is not, then rather use:
```
f"""\
```